master/internal/experiment.go

```diff
@@ -284,8 +284,11 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 
 	// Experiment shutdown logic.
 	case actor.PostStop:
-		if err := e.db.SaveExperimentProgress(e.ID, nil); err != nil {
-			ctx.Log().Error(err)
+		if e.State == model.CompletedState || e.State == model.StoppingCompletedState {
+			setProgress := 1.0
+			if err := e.db.SaveExperimentProgress(e.ID, &setProgress); err != nil {
```

Comment:
fyi there is a `ptrs` package in determined for this quirk. so you can do `ptrs.FloatPtr(1.0)`.

Reply:
✅  oh! ok I'll use that here and in the future